### PR TITLE
Feature: Add black and white logos as inline SVGs

### PR DESCRIPTION
**Pull request by paulathevalley**
- Save a network request by converting the logo in the footer to an inline SVG (thanks @divmain!)
- Add an option to use the white logo 
- Add a unique `key` to each link so Radium can find them when adding link style overrides 

I tested these with `npm link` in the spectacle-docs repo.

/cc @coopy @tptee

### CODE REVIEW

File: src/components/footer.jsx [hunk 1]
```diff
@@ -42,18 +44,23 @@ class Footer extends React.Component {
           this.props.styleOverrides && footerStyles.styleOverrides
         ]}>
         <span style={[footerStyles.text]}>
-          Made with love in Seattle by
```
Comment (coopy) [inline src/components/footer.jsx]: We're dropping :space_invader: :syringe:?

**Comment by coopy** [comment]
LGTM, if Marketing (cc @elizagrace) wants to change the tagline and drop `in Seattle` ;(

**Comment by paulathevalley** [comment]
@elizagrace mentioned dropping "in Seattle" for our pitch deck, so I suspected it made sense to keep it consistent here. I’ll allow her to confirm/reject this change!

**Comment by elizagrace** [comment]
I suggested we remove "in seattle" because not all of our products are made here. anything ken makes is   "in new jersey".

**Pull request merged**